montblanc/impl/biro/v2/loaders/loaders.py

```diff
@@ -23,6 +23,10 @@
 import montblanc
 import montblanc.impl.common.loaders
 
+ORDER_CASA = 'casa'
+ORDER_OTHER = 'other'
+VALID_ORDERINGS = [ORDER_CASA, ORDER_OTHER]
```

Comment:
Its better to use a string constants, because the constant can be used throughout the code base. Then, if the string needs to change, one doesn't need to refactor large amounts of code.